packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -215,12 +286,29 @@ function SideNavGroup({
                   opacity: 1;
                   max-height: ${ulRef?.current?.getBoundingClientRect()
                     .height}px;
+                  border-bottom: 1px solid ${uiColors.gray.light2};
                 `]: state === 'entered',
                 [transitionStyles.exiting]: state === 'exiting',
                 [transitionStyles.exited]: state === 'exited',
               })}
             >
-              <ul ref={ulRef} role="menu" className={ulStyleOverrides}>
+              <ul
+                ref={ulRef}
+                id={menuId}
+                aria-labelledby={menuGroupLabelId}
+                className={cx(
```

Comment:
Should we use prefersReducedMotion to disable these transitions when necessary

Reply:
Transitions that don't use motion are fine with the setting, including opacity. 

Example: Turn on "reduce motion", and then open mission control on your mac (three finger swipe up via trackpad, or the f3 key). They used to transform and scale the windows, but switch to doing an opacity transition.